src/Collection.js

```diff
@@ -753,6 +803,68 @@ Collection.prototype.scroll = function (scrollId, options, filters, cb) {
 };
 
 /**
+ * Retrieves next result of a search with scroll query.
+ *
+ * @param {string} scrollId
+ * @param {object} [options] - Optional parameters
+ * @param {responseCallback} cb - Handles the query response
+ */
+Collection.prototype.scrollSpecifications = function (scrollId, options, cb) {
```

Comment:
Other scroll methods return a KuzzleSearchResult object, shouldn't this one do the same?